### PR TITLE
Inspect code with Goland static analysis tools

### PR DESCRIPTION
**Pull request by brianstrauch**
Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----
* Deleted dead public functions and constants
* Code style suggestions (SNAKE_CASE to CamelCase, etc.)
* Remove redundancies and standardize function headers

Test & Review
-------------
Tests still pass

### CODE REVIEW

**Review by Guttz — APPROVED** [review]

File: internal/flink/command.go [hunk 1]
```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/config"
 )
 
+const serviceAccountWarning = "[WARN] No service account provided. To ensure that your statements run continuously, " +
```
Comment (MuweiHe) [inline internal/flink/command.go:13]: we may want to make those style more consistent. in your previous pr it was using `[WARNING]`

Comment (brianstrauch) [reply internal/flink/command.go:13]: Good point, changing the other PR to use `[WARN]`! (The goal is to match the style of the logging output)

File: internal/kafka/command_topic_update.go [hunk 2]
```diff
@@ -19,10 +19,7 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/types"
 )
 
-const (
-	updateTopicConfigRestMsg    = "Updated the following configuration values for topic \"%s\"%s:\n"
-	readOnlyConfigNotUpdatedMsg = "(read-only configs were not updated)"
-)
+const updateTopicConfigRestMsg = "Updated the following configuration values for topic \"%s\"%s:\n"
```
Comment (MuweiHe) [inline internal/kafka/command_topic_update.go:22]: is this message being used as error or suggestion? it's capitalized.

Comment (brianstrauch) [reply internal/kafka/command_topic_update.go:22]: Looks like it's only used in one place though... updating!

**Review by MuweiHe — APPROVED** [review]
Thank you! :)

**Pull request merged**